src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -44,15 +45,54 @@ For example, if your app is already installed:
 
 <div class="w-clearfix"></div>
 
-### Supported app types you can check
+To use the `getInstalledRelatedApps()` API, you need to tell your app about
+your site, then tell your site about your app. Once you've defined the
+relationship between the two, you can check if the app is installed.
 
-Your website can check if your:
+### Supported app types you can check
 
-* [Android app](#check-android) is installed
-* [Windows (UWP) app](#check-windows) is installed
-* Progressive Web App is installed, if it's running in the
-  [same scope](#check-pwa-in-scope) or in a
-  [different scope](#check-pwa-out-of-scope)
+<div class="w-table-wrapper">
+  <table class="w-table--top-align">
+    <thead>
+      <tr>
+        <th>Can check for your</th>
+        <th>Can check from</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <a href="#check-android">Android app</a>
+        </td>
+        <td>
+          Android only<br>
+          Chrome 80 or later
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="#check-windows">Windows (UWP) app</a>
```

Comment:
UWP?

Reply:
Technical term, Universal Windows Program - I think... Its how they refer to that type of app.